internal/coreinternal/processor/filterconfig/config.go

```diff
@@ -130,8 +136,13 @@ func (mp *MatchProperties) ValidateForSpans() error {
 		return errors.New("log_severity_texts should not be specified for trace spans")
 	}
 
+	if mp.SpanDuration != nil {
+		if mp.SpanDuration.Operator != ">" && mp.SpanDuration.Operator != "<" {
+			return errors.New("span_duration operator should be >, <, or =")
```

Comment:
I might be reading it wrong, but isn't it missing the "=" operator from the conditional? Looking further, looks like the possible operators are ">=", "<=", ">" and "<".